src/custom-elements/indicator.vue

```diff
@@ -11,35 +11,33 @@
 export default {
   name: 'overlay-indicator',
   props: {
-    'overlayIndicatorPackageType': {
-      type: String
+    overlayIndicatorPackageType: {
+      type: String,
     },
-    'overlayIndicatorPackageName': {
-      type: String
+    overlayIndicatorPackageName: {
+      type: String,
     },
   },
   data() {
-    return {}
+    return {};
   },
   computed: {
     store() {
-      return window.__overlay_global_store || {}
+      return window.__overlay_global_store || {};
     },
     packageId() {
-      return `${this.overlayIndicatorPackageType}/${this.overlayIndicatorPackageName}`
+      return `${this.overlayIndicatorPackageType}/${this.overlayIndicatorPackageName}`;
     },
     package() {
-      return this.store.packages[this.packageId] || {};
-    }
+      return this.store.packages?.[this.packageId] || {};
```

Comment:
Added `?.`.

For a reason, the `this.store.packages` is `null`/`undefined`, so it throws an error.

We will find the problem and fix it later.